code/modules/organs/subtypes/augment.dm

```diff
@@ -144,6 +144,18 @@
 	parent_organ = BP_L_HAND
 	aug_slot = slot_l_hand
 
+/obj/item/organ/internal/augment/tool/pen
```

Comment:
This type will have `parent_organ = BP_CHEST` due to not overriding that from `/obj/item/organ/internal/augment`